eclipse-collections-code-generator/src/main/resources/test/set/mutable/boxedMutableHashSetTest.stg

```diff
@@ -101,9 +101,9 @@ public class BoxedMutable<name>SetTest
     public void remove()
     {
         BoxedMutable<name>Set set = this.classUnderTest();
-        Verify.assertTrue(set.remove(<(literal.(type))("1")>));
-        Verify.assertFalse(set.remove(<(literal.(type))("1")>));
-        Verify.assertFalse(set.remove("abc"));
+        assertTrue(set.remove(<(literal.(type))("1")>));
```

Comment:
No point to do via Verify

Reply:
There might be a lot of examples like this. It's nice to clean them up and maybe we can come back and do another PR that gets more of them. IntelliJ can show them with a warning, because it's accessing a static method via a subclass. However, we have to temporarily configure IntelliJ to treat generated code as non-generated, or the IDE will suppress most warnings.